.github/workflows/ci.yaml

```diff
@@ -27,10 +27,10 @@ jobs:
       fail-fast: false
       matrix:
         env:
-          - KEYCLOAK_VERSION: 18.0.2
```

Comment:
Btw. I think we should keep Keycloak 18.0.2 if possible as this is AFAIK the last version that is compatible with the latest(?) RH-SSO release.

Reply:
Is there still a high usage or any other reason not to use older compatible versions of the cli?
I would prefer to keep the maintenance with three versions only.

Reply:
AFAIK every company that uses an up-to-date RH-SSO deployment is effectively on API level 18.0.x.
https://access.redhat.com/articles/2342881

We have a few customers who use RH-SSO, but the vast majority uses Keycloak OSS.

If version 18 is dropped, then new versions of keycloak-config-cli cannot be used with RH-SSO (based on the old Keycloak versions), due too incompatible representations.

I don't know how long RH-SSO stays on the old WF/JBoss EAP based Keycloak, but until then those users can only use older keycloak-cli-versions.

How about:
- we drop support 18 now, and wait for users to complain via reporting issues. 
- Then we could offer to provie "bug-fix" releases for the 5.6.x, which is the last version with Keycloak 18 support.

Reply:
alernatively ... we could just try to keep support for 18.0.2 (just until 23.x comes out) to give users more time to upgrade.

Reply:
Decision: we keep support for 18.0.x for now.